napari/_tests/test_notebook_display.py

```diff
@@ -11,8 +11,11 @@ def test_nbscreenshot(make_napari_viewer):
     data = np.random.random((10, 15))
     viewer.add_image(data)
 
-    rich_display_object = nbscreenshot(viewer)
+    rich_display_object = nbscreenshot(viewer, alt_text="My alt text")
```

Comment:
And maybe try to put some naughty things in here like, `` <>"'`& `` you can get inspirations from https://github.com/minimaxir/big-list-of-naughty-strings.